cmd/cmd_pkg.go

```diff
@@ -219,9 +223,53 @@ func loadCommandHandler(cmd *cobra.Command, args []string) {
 	buildImageFlags := NewBuildImageCommandFlags(flags)
 	runLocalInstanceFlags := NewRunLocalInstanceCommandFlags(flags)
 	pkgFlags := NewPkgCommandFlags(flags)
-
 	pkgFlags.Package = args[0]
 
+	for _, port := range runLocalInstanceFlags.Ports {
```

Comment:
Let's do this in `flags_run_local_instance` to cover all the cases where the instances are launched locally (https://github.com/nanovms/ops/blob/master/cmd/flags_run_local_instance.go#L35).

Reply:
Fixed by https://github.com/nanovms/ops/pull/1025/commits/991df4ea689ffaeab24358c7c00c80f62c20f7b2